tools/toitp.toit

```diff
@@ -132,23 +114,29 @@ print_senders program bc:
 filter := ""
 
 main args:
-  parser := ArgumentParser
-  parser.describe_rest ["snapshot-file", "[filter]"]
-  parser.add_flag "help"            --short="h"
-  parser.add_flag "literals"        --short="l"
-  parser.add_option "literal"
-  parser.add_flag "classes"         --short="c"
-  parser.add_flag "dispatch_table"  --short="d"
-  parser.add_flag "method_table"    --short="m"
-  parser.add_flag "method_sizes"
-  parser.add_flag "bytecodes"       --short="bc"
-  parser.add_flag "senders"         --short="s"
-  parser.add_flag "primitive_table" --short="p"
-
-  parsed := parser.parse args
-  if parsed["help"]: usage --exit_code=0
-  if parsed.rest.size > 1: filter = parsed.rest[1]
-  snapshot := SnapshotBundle.from_file parsed.rest[0]
+  parsed := null
+  parser := cli.Command "toitp"
+      --rest=[
+          cli.OptionString "snapshot-file" --required,
```

Comment:
as before. Use the `--type`.
For example:
```suggestion
          cli.OptionString "snapshot" --type="file" --required,
```
Don't forget to update the use in line 139.